src/api/client/cache.ts

```diff
@@ -10,13 +10,14 @@ const getVideoKeyArgs = (args: Record<string, GetVideosQueryVariables['where']>
   const isPublic = args?.where?.isPublic_eq
   const channelIdIn = args?.where?.channelId_in ? JSON.stringify(args.where.channelId_in) : ''
   const createdAtGte = args?.where?.createdAt_gte ? JSON.stringify(args.where.createdAt_gte) : ''
+  const idIn = args?.where?.id_in ? JSON.stringify(args.where.id_in) : ''
```

Comment:
Why do we need that?

Reply:
We don't need that anymore, forgot to revert this